models/repo_list_test.go

```diff
@@ -32,23 +30,20 @@ func TestSearchRepositoryByName(t *testing.T) {
 		Keyword:  "test_repo",
 		Page:     1,
 		PageSize: 10,
-		Searcher: nil,
 	})
 
-	assert.NotNil(t, repos)
 	assert.NoError(t, err)
 	assert.Equal(t, int64(2), count)
+	assert.Len(t, repos, 2)
 
 	// test search private repository on explore page
 	repos, count, err = SearchRepositoryByName(&SearchRepoOptions{
 		Keyword:  "repo_13",
 		Page:     1,
 		PageSize: 10,
 		Private:  true,
-		Searcher: &User{ID: 14},
```

Comment:
IMO you should leave it as it is.
Although the implementation doesn't need it because no OwnerID i set, but it could be a test that it has no impact.

Reply:
ok